frontend/app/services/letters-service.server.ts

```diff
@@ -1,8 +1,11 @@
+import moize from 'moize';
 import { z } from 'zod';
 
 import { getEnv } from '~/utils/env.server';
 import { getLogger } from '~/utils/logging.server';
 
+const log = getLogger('letters-service.server');
```

Comment:
```suggestion
const logger = getLogger('letters-service.server');
```

Reply:
GTFO with your spring boot example... it's because `log.error` should be `logger.logError` in reality.